docs/deployment/README.md

```diff
@@ -191,6 +191,31 @@ The same information can be obtained from the AWS CLI command: `aws iam list-rol
 
 The `iam` deployment also creates an instance profile named `<stack-name>-ecs` that can be viewed from the AWS CLI command: `aws iam list-instance-profiles`.
 
+--------------
+## Configure and Deploy the Database stack
```

Comment:
Fair point. Perhaps it would be better to have one "configuring" section and three "deploying" sections. I had some trouble as there are a lot of references to template-deploy, which will also need to be updated.